spynnaker/pyNN/extra_algorithms/splitter_components/splitter_abstract_pop_vertex_neurons_synapses.py

```diff
@@ -68,6 +68,9 @@
 # fit in DTCM (14-bits = 16,384 16-bit ring buffer entries = 32Kb DTCM
 MAX_RING_BUFFER_BITS = 14
 
+# The maximum number of cores to consider acceptable for a single chip
+_MAX_CORES = 16
```

Comment:
Would be better if this came from somewhere that knew if ChipPowerMonitors where taking one chip everywhere

Reply:
I have changed this to 15 for more safety.  This is the maximum number of cores that should be used per chip, when also accounting for poisson cores.  This avoids issues with running some demos where a large number of One-to-one Poisson sources target a single population, which would otherwise fail as it would try to put them all on a single chip!  This was then additionally made to check against the synapse cores requested.